internal/archive/archive.go

```diff
@@ -53,6 +53,12 @@ var bulkClient = &http.Client{
 
 var bulkDo = bulkClient.Do
 
+// SetTimeout sets the Timeout for each request.
+// A Timeout of zero means no timeout.
+func SetTimeout(timeout time.Duration) {
```

Comment:
`SetTimeout` or `setTimeout`? the latter seems more consistent with existing functions

Reply:
`SetTimeout`. The function will need to be public to be accessible from other packages.